CDP4-Server.sln.DotSettings

```diff
@@ -249,5 +249,10 @@
 	<s:Boolean x:Key="/Default/CodeStyle/Naming/CSharpNaming/ApplyAutoDetectedRules/@EntryValue">False</s:Boolean>
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=PrivateInstanceFields/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="aaBb" /&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=PrivateStaticFields/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="aaBb" /&gt;</s:String>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ECSharpKeepExistingMigration/@EntryIndexedValue">True</s:Boolean>
```

Comment:
i hope we are not going to change the settings on each commit? i saw in an earlier commit that the coding conventions had been commited and agreed, why already a change? or was the proposed settings file not agreed on?

Reply:
It was agreed on.
I didn't expect to see these changes too. They were not made on purpose.
I did an undo on these changes before checkin, but somehow they were checked in anyway.
Strangely, after the "undo" action, Visual Studio still showed the DotSettings file in its "Changed" list, but the diff didn't show any changes on it.

I hope this was a one time incident.